docs/index.rst

```diff
@@ -36,7 +36,7 @@ If you are new to the concept of smart contracts we recommend you start with
 :ref:`an example smart contract <simple-smart-contract>` written
 in Solidity. When you are ready for more detail, we recommend you read the
 :doc:`"Solidity by Example" <solidity-by-example>` and
-:doc:`"Solidity in Depth" <solidity-in-depth>` sections to learn the core concepts of the language.
+"Language Description" sections to learn the core concepts of the language.
```

Comment:
Had to change this like because "Solidity in Depth" cannot be linked to anymore.